src/pollParams.js

```diff
@@ -0,0 +1,13 @@
+var inquirer = require('inquirer');
+
+module.exports = (questions) => new Promise((resolve, reject) => {
+  if (!questions) {
+    return resolve({});
+  }
+
+  try {
+    inquirer.prompt(questions, resolve);
+  } catch (e) {
+    reject(e);
```

Comment:
I don't think we need a try/catch in a promise, if anything throws here, it will be passed to reject resolving function by default.